### PR TITLE
[Ready for Review] Spanish - Introduction to Web Accessibility

### PR DESCRIPTION
**Pull request by visanju**
Applied changes and one typo solved

### CODE REVIEW

**Comment by remibetin** [comment]
Hi @visanju,

Are you still working on this PR or is it ready for review?

**Review by cmuncharaz — APPROVED** [review]

File: content/index.es.md [hunk 1]
```diff
@@ -19,6 +20,12 @@ github:
   branch: gh-pages
   path: content/index.es.md   # Add the language shortcode to the middle of the filename, for example index.fr.md
 
+permalink: /fundamentals/accessibility-intro/es   # Add the language shortcode to the end, with no slash at the end. For example /path/to/file/fr
```
Comment (remibetin) [inline content/index.es.md:21]: Permalink, ref, changelog and layout attributes have been duplicated (see line 14 to 17 & line 23 to 27).

Please only keep lines 23 to 27.

File: content/index.es.md [hunk 2]
```diff
@@ -173,8 +180,8 @@ Es más eficiente y efectivo incorporar la accesibilidad desde el principio de l
 -   Para entender algunas de las barreras comunes de accesibilidad desde la perspectiva de la evaluación, acceda a [[Pruebas sencillas - Una Primera Revisión]](/test-evaluate/preliminary/).
 -   Para conocer algunas de las consideraciones básicas de diseño, edición y desarrollo para accesibilidad, acceda a [[Consejos Para Empezar]](/tips/).
 -   Cuando esté preparado para saber más sobre desarrollo y diseño, probablemente le serán útiles recursos como:
-    -   [Cómo cumplir las WCAG (Referencia Rápida)](http://www.w3.org/WAI/WCAG21/quickref/)
-    -   [Tutoriales de Accesibilidad Web](https://www.w3.org/WAI/tutorials/)
+    -   [Cómo cumplir las WCAG (Referencia Rápida)](http://www.w3.org/WAI/WCAG22/quickref/)
```
Comment (remibetin) [inline content/index.es.md]:
```suggestion
    -   [Cómo cumplir las WCAG (Referencia Rápida)](https://www.w3.org/WAI/WCAG22/quickref/)
```

HTTPS is now used in the original version.

File: content/index.es.md [hunk 3]
```diff
@@ -209,7 +216,7 @@ Existen herramientas que ayudan en la evaluación. Sin embargo, ninguna herramie
 
 ![Imagen de logotipo; Marcado HTML img alt='Web Accessibility Initiative logo'](https://www.w3.org/WAI/intro/alt-logo.png){:.right}
 
-Las imágenes deben incluir una *[alternativa textual equivalente](http://www.w3.org/TR/UNDERSTANDING-WCAG20/text-equiv.html)* (texto alternativo) en el marcado/código.
+Las imágenes deben incluir una *[alternativa textual equivalente](http://www.w3.org/TR/UNDERSTANDING-WCAG22/text-equiv.html)* (texto alternativo) en el marcado/código.
```
Comment (remibetin) [inline content/index.es.md]:
```suggestion
Las imágenes deben incluir una *[alternativa textual equivalente](https://www.w3.org/WAI/WCAG22/Understanding/text-alternatives)* (texto alternativo) en el marcado/código.
```
The new URL is https://www.w3.org/WAI/WCAG22/Understanding/text-alternatives

**Review by remibetin — CHANGES_REQUESTED** [review]
Hi @visanju,

Thank you for this translation update!

Please see my comments in the "Files changed" tab.

There is also a new bullet point missing in the translation: (in "More Info on Accessibility is Important" box)

```
   -   Examples of the benefits of [making audio and video media accessible](/media/av/) is in the section [Benefits to Organizations](/media/av/users-orgs/#benefits).
```

Can you translate this part and add it to the translation?

Thanks!

**Comment by remibetin** [comment]
@visanju Thanks to [your issue](https://github.com/w3c/wai-intro-accessibility/issues/86), we have just slightly updated the original resource. See [this diff file](https://github.com/w3c/wai-intro-accessibility/commit/32277e7db5dd0f36d607cd800da1b0831ba84814#diff-a4f40b84fbcf1a7939d596dca4939bbae6e56c05ad06a62b719b42693a9baf07)

Would you update your translation accordingly?

**Comment by cmuncharaz** [comment]
I'm trying to push changes (adding a missing bullet point and updating the translation according to the diff file shared above). However, I'm getting an error when pushing.

I think this error could be due to the "Allow edits from maintainers" option deactivated. Could you please make sure this option is checked, @visanju?

**Comment by remibetin** [comment]
@cmuncharaz Thanks for contributing to this translation! This is greatly appreciated.

I have created a dedicated `es` branch containing @visanju's work. Would you open a new PR to update this branch?

**Comment by remibetin** [comment]
Hi @cmuncharaz 

Have you had the chance to see my reply? Let me know if I can help!

**Pull request merged**